cmd/ocitools/validate.go

```diff
@@ -381,6 +386,47 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 	return
 }
 
+func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
+	logrus.Debugf("check linux resources")
+
+	if r.OOMScoreAdj != nil && (int(*r.OOMScoreAdj) < -1000 || int(*r.OOMScoreAdj) > 1000) {
+		msgs = append(msgs, fmt.Sprintf("OOMScoreAdj range is from -1000 to 1000"))
```

Comment:
On 08/18/2016 12:13 AM, W. Trevor King wrote:
> I think we should leverage runtime-spec's JSON Schema <https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/schema/schema-linux.json#L64-L69> for this sort of thing. Validation should validate against the JSON Schema, and then run additional checks that can't be addressed in JSON Schema (like your limit/swap comparison below).
Well, I didn't know about runtime-spec's JSON Schema before...
It seems it has function to validate min and max value.
But I think it's a little inconvenient to validate bundle by two tools as ocitools and this one.
I think ocitools should have this functions when validating bundle.
can we merge this into ocitools?